app/scripts/modules/core/src/header/SpinnakerHeader.css

```diff
@@ -0,0 +1,42 @@
+.SpinnakerHeader {
```

Comment:
I can't remember.  Have we decided on a naming convention for CSS files?  And class names, for that matter.  Curious for your thoughts here.

Reply:
This is react's naming convention for scoping component styles by using the same name. Some people prefer using .rc-<ComponentName>. I personally do not like that. What do you guys think?

Reply:
While I love following accepted conventions, I don't love having to remember which conventions to follow.  Definitely up for a discussion.

Reply:
While I love following accepted conventions, I don't love having to remember which conventions to follow.  Definitely up for a discussion.

Reply:
This looks so weird to me. But if this is what people are doing, might as well get used to it.

Reply:
Waiting on the poll for the verdict!